v22.1/security-reference/transport-layer-security.md

```diff
@@ -0,0 +1,280 @@
+
+In mutually authenticated TLS connections, each party must have a key pair issued by a jointly trusted CA. This is a good system for internal components within a distributed system, and generally works well for long-lived clients.
+
+In one-sided TLS authentication, only the server must have a key pair; the user may be able to access information without any authentication. This works well when:
+
+- The client needs no authentication, for example, for a public read-only website or API.
+- When the user must authenticate with another mechanism, such as a username/password combination or Single Sign-On (SSO).
+
+### Symmetric and asymmetric encryption in TLS
+
+Encryption using a key pair is "asymmetric", in that a different key is used to encrypt and decrypt. Encryption is called "symmetric" where the same key is used to encrypt and decrypt&mdash;as the metal key in a mechanical lock-and-key system is used to both lock and unlock.
+
+Symmetric encryption is more efficient than asymmetric encryiption, but it requires all parties to have the single key shared key prior to the exchange.
+
+With asymmetric encryption, by contrast, each party can share their public key, allowing anyone to send them messages that only they can decrypt with their private key. In the context of internet communication
+
+In the TLS protocol, asymmetric encryption using the public/private key pairs is used to securely exchange random data from that is used to create shared **session keys**, which are then used by both parties to (symmetrically) encrypt and decrypt data for the remainder of the session. Since a session key is created with information shared using asymmetric encryption, and is used only for the duration of a session, TLS combines the best of both asymmetric encryption (the ability to establish an encrypted communication channel without having to have first securely shared a key) and symmetric encryption (computational efficiency).
+
+{{site.data.alerts.callout_info}}
+The generation and management of session keys is fully automated within the TLS protocol. You do not ever need to provision or manage TLS session keys.
+{{site.data.alerts.end}}
+
+## What is Public Key Infrastructure (PKI)?
+
+Encryption is powerful and important, but without identity authentication, it's not very useful: if you don't know who is sending you messages in first place, it's small comfort that nobody *else* is tampering with the contents or eavesdropping on the conversation.
+
+If you encrypt a  message with a TLS public key, you know that only a holder of the matching private key will be able to decrypt it. And perhaps the holder of the public certificate identifies themself as your friend, your bank, your employer, or a government. But how can you trust that the certificate was ever actually held by the party you want to reach, rather than an imposter?
```

Comment:
```suggestion
If you encrypt a message with a public key, you know that only a holder of the matching private key will be able to decrypt it. And perhaps the holder of the public certificate identifies themself as your friend, your bank, your employer, or a government. But how can you trust that the certificate was ever actually held by the party you want to reach, rather than an imposter?
```